src/java/org/apache/cassandra/db/lifecycle/LogFile.java

```diff
@@ -60,7 +60,7 @@
     static String EXT = ".log";
     static char SEP = '_';
     // cc_txn_opname_id.log (where cc is one of the sstable versions defined in BigVersion)
-    static Pattern FILE_REGEX = Pattern.compile(String.format("^(.{2})_txn_(.*)_(.*)%s$", EXT));
+    static Pattern FILE_REGEX = Pattern.compile(String.format("^([a-z]+)_txn_(.*)_(.*)%s$", EXT));
```

Comment:
So this is because you changed sstable metadata version to mdn right?

Reply:
yes